pkgs/fc/qemu/default.nix

```diff
@@ -88,7 +91,7 @@ in
       py.pytest-cov
       py.pytest-timeout
       py.mock
-
+      fc-ceph
```

Comment:
Same as above, this pulls in `fc-qemu` into a KVM host, which is just a ceph client.

Reply:
No, these are the test dependencies. fc-qemu does call those in the test suite.